### ISSUE TITLE
Adding segtype property to Lv breaks code

### ISSUE THREAD

**Issue by tasleson** [tasleson/lvm-dubstep#14]
@vpodzime commit 7bd89975f2df2bff94d4246873cc35b73f91fce4 added segtype .  When we have a ~~mirrored LV~~ simple LV across PVs you get multiple lines of output referring to the same LV (which causes the stacktrace below) when you do a lvs and add `segtype`.  After looking at this some more I believe we need to move this value to the physical devices as a single LV can have multiple segments on 1 or more PVs, each with a different segtype, but I need to look into it some more.

```
Traceback (most recent call last):
  File "./lvmdbus.py", line 1319, in <module>
    load(sys_bus, lvm)
  File "./lvmdbus.py", line 1111, in load
    for l in load_lvs(connection, obj_m):
  File "./lvmdbus.py", line 1083, in load_lvs
    l['segtype'])
  File "./lvmdbus.py", line 693, in __init__
    super(Lv, self).__init__(c, object_path, self.DBUS_INTERFACE, load_lvs)
  File "/home/tasleson/projects/lvm-dubstep/utils.py", line 186, in __init__
    dbus.service.Object.__init__(self, conn, object_path)
  File "/usr/lib/python2.7/site-packages/dbus/service.py", line 604, in __init__
    self.add_to_connection(conn, object_path)
  File "/usr/lib/python2.7/site-packages/dbus/service.py", line 695, in add_to_connection
    self._fallback)
KeyError: "Can't register the object-path handler for '/com/redhat/lvm1/lv/8': there is already a handler"
```

**Comment by vpodzime** [comment]
Hmm, I didn't know about that. Maybe we could, at least for now, just append the segtype values to the same field separated by commas or something?

**Comment by tasleson** [comment]
Example:

```
# lvs -o name,segtype,seg_pe_ranges
  LV    Type    PE Ranges                
  lvol0 linear  /dev/sdd:14-30           
  lvol0 linear  /dev/sde:15-22           
  lvol1 linear  /dev/sdd:0-2             
  lvol1 striped /dev/sdd:3-3 /dev/sde:0-0
```

**Comment by tasleson** [comment]
I'm leaning towards the property to return `as`, array of strings.  If an API user wants to concatenate them they can certainly do that.  To clarify, return the array of unique `segtype`.

**Comment by vpodzime** [comment]
> I'm leaning towards the property to return as, array of strings. If an API user wants to concatenate them they can certainly do that. To clarify, return the array of unique segtype.

Sounds great to me.

**Comment by tasleson** [comment]
Corrected with: 1215c690ee4d022a16a193219592e9321858932d

**Comment by vpodzime** [comment]
Great, thanks!
